lib/cartopy/io/img_tiles.py

```diff
@@ -862,7 +851,7 @@ def _image_url(self, tile):
 
 class LINZMapsTiles(GoogleWTS):
 
-    def __init__(self, apikey, layer_id, api_version="v4",
+    def __init__(self, apikey, layer, api_version="v4",
```

Comment:
This one is called "layer_id" in their documentation. I'm not sure how I feel about changing this.
https://www.linz.govt.nz/guidance/data-service/linz-data-service-guide/changesets/changeset-url-syntax

What if you kept it all the same here and in the super call you call it with `layer=layer_id`